src/validator/GoValidator.cpp

```diff
@@ -63,6 +63,7 @@ Status GoValidator::validateWhere(WhereClause* where) {
         ExpressionUtils::rewriteLabelAttribute<EdgePropertyExpression>(filter_);
     }
 
+    filter_ = where->filter();
     auto typeStatus = deduceExprType(filter_);
```

Comment:
This don't report error when got nullptr ?

Reply:
It won't become nullptr when there is a where clause.